toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -348,33 +350,289 @@ func enableOrDisableServices(services imagecustomizerapi.Services, imageChroot *
 	return nil
 }
 
-func loadOrDisableModules(modules imagecustomizerapi.Modules, imageChroot *safechroot.Chroot) error {
+func loadOrDisableModules(modules []imagecustomizerapi.Module, imageChroot *safechroot.Chroot) error {
```

Comment:
There is now a lot of code for handling kernel modules. It might be worth splitting this off into its own file.

Reply:
good point. Will do